samples/bot-task-module/csharp/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
-  "MicrosoftAppType": "",
-  "MicrosoftAppId": "",
-  "MicrosoftAppPassword": "",
-  "MicrosoftAppTenantId": "",
-  "BaseUrl": "https://YourDeployedBotUrl.com"
-}
+  "MicrosoftAppType": "<<MicrosoftAppType>>",
+  "MicrosoftAppId": "<<MicrosoftAppId>>",
+  "MicrosoftAppPassword": "<<MicrosoftAppPassword>>",
+  "MicrosoftAppTenantId": "<<MicrosoftAppTenantId>>",
+  "BaseUrl": "<<Ngrok>>"
+}
```

Comment:
Update the Readme for these updated placeholders.

Reply:
fixed